ceph_bootstrap/__init__.py

```diff
@@ -89,11 +89,12 @@ def config_shell(config_args):
 @cli.command(name='deploy')
 @click.option('-n', '--non-interactive', is_flag=True, default=False,
               help='Run deploy in non-interactive mode')
-def deploy(non_interactive):
+@click.argument('minion_id', required=False)
```

Comment:
Add an `help` description to the argument

Reply:
Not supported by `@click.argument`